Formula/mupdf-tools.rb

```diff
@@ -25,6 +25,13 @@ class MupdfTools < Formula
     because: "mupdf and mupdf-tools install the same binaries"
 
   def install
+    # temp patch suggested by Robin Watts in https://bugs.ghostscript.com/show_bug.cgi?id=706112#c1
+    if OS.mac?
+      inreplace "source/fitz/encode-basic.c", '#include "z-imp.h"',
+                "#include \"z-imp.h\"\n#include <limits.h>"
+      inreplace "source/fitz/output-ps.c", '#include "z-imp.h"',
+                "#include \"z-imp.h\"\n#include <limits.h>"
+    end
```

Comment:
Since we have [submitted this patch](https://github.com/ArtifexSoftware/mupdf/pull/32), we can just update the PR to reference it.

Reply:
Not sure if I correctly got your idea, but some updates to the comment is pushed.